huaweicloud/services/dds/data_source_huaweicloud_dds_flavors_v3.go

```diff
@@ -115,20 +116,20 @@ func dataSourceDDSFlavorV3Read(_ context.Context, d *schema.ResourceData, meta i
 		flavorList = append(flavorList, flavor)
 	}
 
-	logp.Printf("Extract %d/%d flavors by filters.", len(flavorList), len(allFlavors))
+	log.Printf("Extract %d/%d flavors by filters.", len(flavorList), len(allFlavors))
```

Comment:
logging level is missing

```suggestion
	log.Printf("[DEBUG] extract %d/%d flavors by filters.", len(flavorList), len(allFlavors))
```